vnext/templates/cpp-app/windows/MyApp/MyApp.vcxproj

```diff
@@ -13,6 +13,7 @@
     <DefaultLanguage>en-US</DefaultLanguage>
     <MinimumVisualStudioVersion>17.0</MinimumVisualStudioVersion>
     <AppxPackage>false</AppxPackage>
+    <ReactNativeWindowsVersion>{{ rnwVersion }}</ReactNativeWindowsVersion>
```

Comment:
I beleive that this variable could contain things like prerelease.1.  Which wouldn't work with https://learn.microsoft.com/en-us/visualstudio/msbuild/property-functions?view=vs-2022#msbuild-version-comparison-functions.
Maybe we should stamp a variable with a version that never includes the prerelease part of the build number.  And maybe another variable which includes the full version including the prerelease.

Reply:
This would also apply to canary versions correct?

Reply:
What is the purpose of this> The property for version of RNW being used is already defined in `vnext/PropertySheets/Generated/PackageVersion.g.props`